src/std/crypto/README.org

```diff
@@ -0,0 +1,390 @@
+#+TITLE: Crypto with libcrypto
```

Comment:
this is not really a `libcrypto` tutorial, more of a JWT tutorial. I think we should change the tile and also rename the file. 
Maybe "Using JWTs in Web Applications" or something? filename could be just `jwt.md` (errrr org)